docs/COMPONENT_GUIDELINES.md

```diff
@@ -29,3 +29,80 @@ So solutions like `@media-queries` might not work, and `window` shouldn't be dir
 
 * `wix-ui-tpa` are rendered for Wix's users-of-users, and so they must comply with web accessibility
  principles.
+ 
+ ## Typography
+ When creating a new component that renders text, we should in most cases use the `<Text/>` component with the correct props and style.  
```

Comment:
```suggestion
 When creating a new component that renders text, simply use the `<Text/>` component. It already contains pre-defined style variants. simply follow it's API in the documentation page
```

Reply:
Unfortunately the `Text` element is old and can't be easily extended.
I prefer to use the current solution, and once we fix the Text element we will update this doc